### PR TITLE
Add support for `.eh_frame` unwinding sections

### PR DESCRIPTION
**Pull request by fitzgen**
Not ready to land yet -- going to need to rebase.

Take a look at the code if you want.

Mostly trying to guage whether coverage is going up or down, and see which branches I might have missed unit tests for.

### CODE REVIEW

**Comment by fitzgen** [comment]
... and apparently CI won't run until I resolve the conflicts. Will try and do that tomorrow.

**Comment by fitzgen** [comment]
Ok, rebased! Not as bad as I was afraid of.

**Comment by fitzgen** [comment]
I think this is ready for review, care to take a look @philipc? Thanks!

File: src/parser.rs [hunk 1]
```diff
@@ -697,4 +881,388 @@ mod tests {
...
+        let input = input.get_contents().unwrap();
+        let input = EndianBuf::<LittleEndian>::new(&input);
+
+        assert_eq!(parse_encoded_pointer(encoding, &bases, address_size, input, input),
+                   Err(Error::UnknownPointerEncoding));
+    }
+
+    #[test]
+    fn test_parse_encoded_pointer_aligned() {
+        // FIXME!
```
Comment (philipc) [inline src/parser.rs]: What needs fixing?

Comment (fitzgen) [inline src/parser.rs]: Supporting pointers encoded with `DW_EH_PE_aligned`.

**Comment by fitzgen** [comment]
I'm going to land this now, but this is going to need to undergo a bit more refactoring to deal with `.eh_frame` in the wild (and ealier DWARF versions). I'm working on it.

**Pull request merged**